### DISCUSSION TITLE
Missing files in Albedo Pool

### DISCUSSION THREAD

**Discussion by ackerlar** [esm-tools/esm_tools#1095] [General]
**Describe the bug**
I want to run AWIESM with the T63-COREII setup on Albedo. It appears that some ECHAM and JSBACH files are missing in the pool (at least I could not find them)

```
WARNING: File not found: /albedo/pool/FESOM2/core2/tarfilesT63/input/echam6/T63CORE2_jan_surf.nc
WARNING: File not found: /albedo/pool/FESOM2/core2/tarfilesT63/input/echam6/T63CORE2_VGRATCLIM.nc
WARNING: File not found: /albedo/pool/FESOM2/core2/tarfilesT63/input/echam6/T63CORE2_VLTCLIM.nc
WARNING: File not found: /albedo/pool/FESOM2/core2/tarfilesT63/input/jsbach/jsbach_T63CORE2_11tiles_5layers_1850.nc
```

I could not find the whole `tarfilesT63` directory

**System (please complete the following information):**
 - Supercomputer: [albedo]
 - Version [version 6.24.0]

**Comment by nwieters** [comment]
I found some paths in a test runscript for AWIESM:
```
/albedo/pool/FESOM/meshes_default/core/tarfilesT63/input/echam6/T63CORE2_jan_surf.nc
/albedo/pool/FESOM/meshes_default/core/tarfilesT63/input/echam6/T63CORE2_VGRATCLIM.nc
/albedo/pool/FESOM/meshes_default/core/tarfilesT63/input/echam6/T63CORE2_VLTCLIM.nc
/albedo/pool/FESOM/meshes_default/core/tarfilesT63/input/jsbach/jsbach_T63CORE2_11tiles_5layers_1850.nc
```
Are these the files you are looking for? I don't know whether they are specific to the FESOM version.

**Comment by nwieters** [comment]
I think one task is to create an example runscript for awiesm on albedo.

**Comment by ackerlar** [comment]
thanks @nwieters! I used now your path for the JSBACH and ECHAM files but `/albedo/pool/FESOM2/core2/` for the mesh and it runs. But I guess it is not supposed to be like this. 
I could not use your path for the mesh because the names of the subdirectories (`dist/288p` instead of `dist_288`) and files (`my_list0233.out` instead of `my_list00233.out`) are different to the FESOM2 mesh directories

**Comment by mandresm** [comment]
Hi Lars, I think that the `FESOM/meshes_default/core` are meshes for FESOM-1.4. That would explain why you cannot use it with FESOM2. The FESOM pools and the pools in general are difficult as you know... this is one of the difficulties: FESOM sometimes means FESOM 1 in the pools. So fast answer, in Albedo the pool that guarantees FESOM2 compatibility is called FESOM2

**Comment by nwieters** [comment]
Hi @ackerlar , @mandresm ,
I will convert this into a discussion so that the general issue about the pool an albedo can still be discussed.

**Comment by pgierz** [comment]
Hi there, I am going to be looking into organizing the albedo pool in a better way soon. I'll post an announcement here (or on the MOTD of the machine).

**Reply by pgierz** [reply]
@mandresm: `git-annex` ;-) it is the way forward.
